File: code/scala_sample/src/main/scala/com/buaa/financedata/sample/scala/Count.scala
package com.buaa.financedata.sample.scala

import org.apache.spark._ 
import org.apache.spark.SparkContext._

object Count{
	def main(args: Array[String]){
		val conf = new SparkConf().setAppName("Count")
		val sc = new SparkContext(conf)
		val rdd = sc.textFile("test/TAQ")
		println(rdd.count())
	}
}

File: code/java_sample/src/main/java/com/financedata/sample/java/Count.java
package com.buaa.financedata.sample.java;

import org.apache.spark.SparkConf;
import org.apache.spark.api.java.*;

public class Count{
	public static void main(String[] args){
		SparkConf conf = new SparkConf().setAppName("Count");
		JavaSparkContext sc = new JavaSparkContext(conf);
		JavaRDD<String> input = sc.textFile("test/TAQ");
		System.out.println(input.count());
	}
}
